master/docs/manual/customization.rst

```diff
@@ -567,6 +571,21 @@ See :class:`buildbot.worker.ec2.EC2LatentWorker` for an example.
         If ``fast`` is ``True`` then the function should call back as soon as it is safe to do so, as, for example, the master may be shutting down.
         The value returned by the callback is ignored.
 
+    .. py:attribute:: builds_may_be_incompatible
+
+        Determines if new instances have qualities dependent on the build.
+        If ``True``, the master will call ``is_compatible_with_build`` to determine whether new builds are compatible with the started instance.
+        By default, this is ``False``.
+
+    .. py:method:: is_compatible_with_build(self, build_props)
```

Comment:
we follow the twisted API naming scheme, so that would be isCompatibleWithBuild()

Reply:
OK, I got confused by `start_instance` and `stop_instance`.

Reply:
yes, this practice is not coherent :-(

Reply:
Fixed now.